packaging/tests/tas-restapi/pom.xml

```diff
@@ -184,9 +176,13 @@
        </dependency>
 
         <dependency>
-            <groupId>org.glassfish</groupId>
-            <artifactId>jakarta.json</artifactId>
-            <scope>test</scope>
+            <groupId>jakarta.json</groupId>
```

Comment:
api might be not needed here. The implementation should be in a `test` scope as before.